peps/pep-0772.rst

```diff
@@ -0,0 +1,455 @@
+* Increased tactical and fundraising support from the PSF, to increase capacity
+  and funding available to packaging tools.
+
+=============
+Specification
+=============
+
+-----------------
+Packaging Council
+-----------------
+
+The Packaging Council will be composed of 5 individuals.
+
+Mandate
+=======
+
+The council shall work to:
+
+* Improve Python packaging's user experience.
+* Maintain the quality and stability of the Python packaging standards.
+* Formalise and maintain the relationship with the core team as well as the
+  PSF.
+* Establish appropriate decision-making processes.
+* Seek consensus among contributors before acting in a formal capacity.
+* Make contributing as accessible, inclusive, and sustainable as possible.
+
+Responsibilities
+================
+
+The council shall:
+
+* have broad authority over the Python packaging standards and Python Packaging
+  User Guide, that are maintained on packaging.python.org.
+* establish processes for making binding decisions regarding packaging
+  standards, tools and implementations as well as for considering
+  ecosystem-wide changes.
+* look for ways to use their direct authority as little as possible – seeking
+  consensus/consent over voting.
+
+To use its powers, the council votes. Every council member must either vote or
+explicitly abstain. Members with conflicts of interest on a particular vote
+must abstain. Passing requires support from a majority of non-abstaining
+council members.
+
+Whenever possible, the council is expected to share its decisions and processes
+publicly in a timely fashion.
+
+Delegations
+-----------
+
+The Python Steering Council would be expected to delegate decision making to
+the Packaging Council for PEPs related to the Python packaging.  The bodies
+would work together on issues that intersect the packaging domain and language
+stewardship (including the CPython implementation, standard library, and
+distribution).
+
+The PSF Board would be encouraged to formally deprecate the Packaging Workgroup
+and the Packaging Council would wholly take on the responsibilities of the
+PSF's Packaging Workgroup.
+
+The PyPA would be expected to work with the Packaging Council to establish a
+decision making process that governs the technical projects under the PyPA
+umbrella.
+
+Processes
+=========
+
+Election of the council
+-----------------------
+
+A council election consists of two phases:
+
+* Phase 1: A member of the Packaging community (defined later in this document)
+  can self-nominate themselves for the council elections.
+* Phase 2: Each core team member can assign zero to five stars to each
+  candidate. Voting is performed anonymously. The outcome of the vote is
+  determined using the `STAR voting system`_, modified to use the `Multi-winner
+  Bloc STAR`_ approach. If a tie occurs, it may be resolved by mutual agreement
+  among the candidates, or else the winner will be chosen at random.
```

Comment:
Is this "core team" the core developers/committers? If not, could you disambiguate? If so, it seems a bit surprising--would it be better to have the packaging community members form the electorate?

Reply:
Whoops! That's a copy-paste mistake.